_data/sidebars/lb4_sidebar.yml

```diff
@@ -15,15 +15,19 @@ children:
   url: Installation.html
   output: 'web, pdf'
 
-- title: 'Getting started'
-  url: Getting-started.html
+- title: 'Examples and tutorials'
+  url: Examples-and-tutorials.html
   output: 'web, pdf'
   children:
 
-  - title: 'Examples and tutorials'
-    url: Examples-and-tutorials.html
+  - title: 'Hello World!'
+    url: Hello-World.html
     output: 'web, pdf'
 
+  - title: 'Getting Started'
```

Comment:
I'd actually prefer Getting Started being at the top level.  That's usually the first place to look when people are starting out.